wp-content/themes/chrisdinh-photography/resources/views/components/slider.blade.php

```diff
@@ -1,27 +1,53 @@
+<!--front-end view of the slider component-->
+<!--run yarn build/dev before coding to implement new tailwind code-->
 @unless ( empty($data) )
-    <section id="{{ $sliderSettings['slider__id'] }}"  class="slider splide relative block w-screen h-screen"
+    <!--parent container of the slider -->
+    <!--uses splidejs.com, read the documentation-->
+    <section id="{{ $sliderSettings['slider__id'] }}"  class="relative w-screen h-screen splider splide"
         data-type="{{ $sliderSettings['slider__type'] }}"
         data-speed="{{ implode(',', $sliderSettings['slider__speed'])  }}"
         data-width="{{ implode(',', $sliderSettings['slider__width']) }}"
         data-height="{{ implode(',', $sliderSettings['slider__height']) }}"
         data-per-page="{{ implode(',', $sliderSettings['slider__per-page']) }}"
         data-per-move="{{ implode(',', $sliderSettings['slider__per-move']) }}"
         data-gap="{{ implode(',', $sliderSettings['slider__gap']) }}"
-        data-arrows="{{ implode(',', $sliderSettings['slider__arrows']) }}"
-        data-pagination="{{ implode(',', $sliderSettings['slider__pagination']) }}"
+        @if (get_field('slider__show-arrows'))
+            data-arrows="{{ implode(',', $sliderSettings['slider__arrows']) }}"
+        @endif
+        @if (get_field('slider__show-pagination'))
```

Comment:
Same thing here, use `$sliderSettings` instead

Reply:
updated in commit 7a8dd48d77f9683e02904cbb06e5dc28970869e9